<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pic2Vid</class>
 <widget class="QWidget" name="Pic2Vid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>591</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="PicDirButton">
      <property name="text">
       <string>select Filep path</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="PicDirPath"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>591</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="VideoDirButton">
      <property name="text">
       <string>select output file path</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="VideoDirPath"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="VidTypeButton">
      <property name="text">
       <string>output format</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="VideoType">
      <item>
       <property name="text">
        <string>avi</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>mp4</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>160</y>
     <width>152</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="FrameButton">
      <property name="text">
       <string>frame rate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="FPS">
      <property name="text">
       <string>30</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>290</width>
     <height>51</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>420</y>
     <width>295</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="ConfirmButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ResetButton">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="CloseButton">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QPushButton" name="VideoNameButton">
      <property name="text">
       <string>output name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="VideoName">
      <property name="text">
       <string>Video</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Frame to Video</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>210</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Here display:
picture names</string>
   </property>
  </widget>
  <widget class="QListWidget" name="PicList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>461</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
